x/data/server/features/msg_define_resolver.feature

```diff
@@ -15,6 +15,11 @@ Feature: Msg/DefineResolver
       When alice attempts to define a resolver with url "https://foo.bar"
       Then expect the error "a resolver with the same URL and manager already exists: unique key violation"
 
+    Scenario: public resolvers can only be defined once per URL
+      Given a public resolver is defined for the url "ipfs:"
+      When alice attempts to define a public resolver with url "ipfs:"
+      Then expect the error "a resolver with the same URL and manager already exists: unique key violation"
```

Comment:
@clevinson I added a test here which demonstrates that a public resolver URL can only be defined once.